### DISCUSSION TITLE
The Coilgun harpoon problems

### DISCUSSION THREAD

**Discussion by SAYtan47** [FakeFishGames/Barotrauma#15290] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The coilgun harpoon ammo box right now holds 4 shots but the counter shows like it has only 3 and there is no visual difference between the bow with 1 shot left and an empty box which can cause confusion. This was tested in sub redactor by spawning items and I didnt test this issue in the campaign so I dont know if its exclusive to the redactor or not.

### Reproduction steps

https://github.com/user-attachments/assets/3c44427a-64a6-4cf2-ade3-eab8da43c2be

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

This has been addressed in our private repo now (https://github.com/FakeFishGames/Barotrauma-development/pull/6153), fix coming up in a future build.
